hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -250,13 +250,9 @@
     }
     {
       name: stub_cpu_mode
-      en_run_modes: ["gen_otp_images_mode"]
-      run_opts: ["+stub_cpu=1"]
-    }
-    {
-      name: stub_cpu_mode_with_test_rom
       en_run_modes: ["sw_test_mode_common"]
-      // The tests using this mode only require the ROM init check to succeed.
+      // Note that the real hardware will always have a valid ROM image, and some stub CPU
```

Comment:
I think it will be good to expand on this and explain that the CPU is still, not going to fetch and execute anything in this mode since it is effectively disconnected.